gallery/src/styles/NxCounter/NxCounterExample.html

```diff
@@ -6,10 +6,12 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="gallery-counter-example-container"><span>Default</span> <span class="nx-counter">42</span></div>
+<div class="gallery-counter-example-container">
+  <span>Default</span><span class="nx-counter">42</span>
+</div>
 
 <div class="gallery-counter-example-container">
-  <span>Active</span> <span class="nx-counter nx-counter--active">2 of 2</span>
+  <span>Active</span><span class="nx-counter nx-counter--active">2 of 2</span>
```

Comment:
I made this change on master.  Did you merge master directly into this branch after merging the v2 feature branch into this branch?  Now that this branch is based off of the feature branch, you should merge from the feature branch to get upstream changes, not directly from master.  To get this out of the PR diff, merge master into the feature branch, and then the feature branch into this branch.

Reply:
> I made this change on master. Did you merge master directly into this branch after merging the v2 feature branch into this branch? Now that this branch is based off of the feature branch, you should merge from the feature branch to get upstream changes, not directly from master. To get this out of the PR diff, merge master into the feature branch, and then the feature branch into this branch.

This morning I merged Master into this branch, then we decided that we were going to put this branch against V2, so this afternoon I merged Master in V2, then V2 into this branch, so I'm not sure why it's showing up.

Reply:
> I merged Master in V2, then V2 into this branch

I just did this part again, should fix it.  I also fixed the version numbers while at it.